config/config.py

```diff
@@ -7,19 +7,28 @@ def playerIcons(poi):
         return "Last known location for {}".format(poi['EntityId'])
 
 
-# Only render the signs with the filter string (stored in environment variable
-# RENDER_SIGNS_FILTER) in them. If RENDER_SIGNS_FILTER is blank or unset,
-# render all signs. RENDER_SIGNS_FILTER defaults to "-- RENDER --" for historic
-# reasons, but also so that people can have secret bases and to keep the render
-# fairly clean.
+# Only render the signs with the filter string in them. If filter string is
+# blank or unset, render all signs. Lines are joined with a configurable string.
 def signFilter(poi):
     # Because of how Overviewer reads this file, we must "import os" again here.
     import os
+    # Only render signs with this function
     if poi['id'] in ['Sign', 'minecraft:sign']:
-        sign_filter_string = os.environ.get('RENDER_SIGNS_FILTER', "")
-        render_all_signs = len(sign_filter_string) == 0
-        if render_all_signs or sign_filter_string in poi.values():
-            return "<br />".join([poi['Text1'], poi['Text2'], poi['Text3'], poi['Text4']])
+        sign_joiner = os.environ.get('RENDER_SIGNS_JOINER', "<br />")
+        sign_filter = os.environ.get('RENDER_SIGNS_FILTER', "")
+        hide_filter = os.environ.get('RENDER_SIGNS_HIDE_FILTER', 'false').lower() == 'true'
+        # Transform the lines into an array.
+        lines = [poi['Text1'], poi['Text2'], poi['Text3'], poi['Text4']]
+        lines = list(map(lambda l: l.strip(), lines))
+        # Determine if we should render this sign
+        render_all_signs = len(sign_filter) == 0
+        render_this_sign = sign_filter.upper() in map(lambda l: l.upper(), lines)
```

Comment:
Maybe this should be an exact test? The previous test was case-sensitive. It works for my use cases, but there might be some use case we aren't thinking of that needs the test to be case-sensitive.

Reply:
That's a fair point. I can keep case sensitive. I was just trying to make it easier for end-users, but I guess it's not terribly hard for them to match the string exactly. I'll tweak that.

Reply:
- Made case sensitive in f5952e7, 
- updated README 7debd46, 
- and made the one other insensitive option case-sensitive to match 7cb43f7